bin/compile

```diff
@@ -24,9 +24,9 @@ export_env_dir() {
 export_env_dir $ENV_DIR
 
 # Build payload
-# { "action": "custom-build-started", "data": { "source_blob": { "version": "xxx" }, "slug": { "commit": "xxx" }, "created_at": "xxx", "updated_at": "xxx" } }
+# { "action": "custom-build-started", "data": { app: { name: "xxx", id: 123 }, "source_blob": { "version": "xxx" }, "slug": { "commit": "xxx" }, "created_at": "xxx", "updated_at": "xxx" } }
 date > /tmp/created_at.date
-payload="{'action':'custom-build-started','data':{'source_blob':{'version':'${SOURCE_VERSION}'},'slug':{'commit':'${SOURCE_VERSION}'},'created_at':'`cat /tmp/created_at.date`','updated_at':'`cat /tmp/created_at.date`'}}"
+payload="{'action':'custom-build-started','data':{'app':{'name':'${HEROKU_APP_NAME}','id':'${HEROKU_APP_ID}'},'source_blob':{'version':'${SOURCE_VERSION}'},'slug':{'commit':'${SOURCE_VERSION}'},'created_at':'`cat /tmp/created_at.date`','updated_at':'`cat /tmp/created_at.date`'}}"
```

Comment:
Call out - this payload is wrong. We don't want app.id we want the equivalent of data.id, where id is the task id.

Reply:
Gotcha - maybe we can tweak the `heroku build running` message then so that it doesn't throw an error if it tries to build the url? That way:
- review apps report the status but don't have a link until `build` is finished
- staging & production always have a link

Reply:
Or even simpler since we know when we are in that state we can just pass a new `skipUrl: true` option and so something like this:
```
    if (skipUrl) {
        originalMessage.elements[0].text = `${message} started at ${formatTime(data.created_at)}`;
    }
    else {
        const url = buildHerokuActivityUrl(data.app, "build");
        originalMessage.elements[0].text = `<${url}|${message}> started at ${formatTime(data.created_at)}`;
    }
```